### PR TITLE
WIP: [feat samuray90] [pug sass javaScript scss css html] project refresher 4

### PR DESCRIPTION
**Pull request by samuray90**

### CODE REVIEW

File: src/_component/connect/connect.pug [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+                p.description-box__paragraph=social.description
+            .description-box__footer
+                a(
+                    href=social.link
+                    target="_blank"
+                ).button__pill--blue
+                    i.icon__link
+                    span.button-text=social.label
+
+.column(span="row" height="10")
```
Comment (marianzburlea) [inline src/_component/connect/connect.pug:42]: you're using 2 spaces tabs for indentation

File: src/_component/connect/connect.pug [hunk 2]
```diff
@@ -0,0 +1,47 @@
+- var sectionTitle = data.connect.sectionTitle
+include ../section-title/section-title.pug
+
+.column(span="row")
```
Comment (marianzburlea) [inline src/_component/connect/connect.pug:4]: You're using 4 spaces indentation

File: src/_component/connect/connect.scss [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+    &::after {
+        content: '';
+        border-radius: 6px;
+        position: absolute;
+        top: 10px;
+        bottom: 10px;
+        right: 10px;
+        left: 10px;
+        background-size: cover;
+        background-image: url('asset/image/poza.jpg');
```
Comment (marianzburlea) [inline src/_component/connect/connect.scss:120]: use **baseUrl** to assure it will work on GitHub as well.

File: src/_data/about-me.json [hunk 4]
```diff
@@ -0,0 +1,12 @@
+{
+    "sectionTitle": "Ionut Daniel Petrut",
+    "mainTitle": "Curriculum Vitae",
+    "subTitle": "Junior Web Developer",
+    "leadTitle": "About me",
+    "descriptionList": [
+        "Describe yourself here erfuhnerhfdwersjerwjwerjhfdwerhjfsdrwejfdswrejhfdjfsdhjfsdjfsd",
```
Comment (marianzburlea) [inline src/_data/about-me.json:7]: add proper content

**Review by marianzburlea — CHANGES_REQUESTED** [review]

File: src/_data/about-me.json (hunk 4 above)
Comment (samuray90) [reply src/_data/about-me.json]: replaced with something more professional.

**Review by marianzburlea — APPROVED** [review]

**Review by ValentinaVanu — APPROVED** [review]

**Review by marianzburlea — APPROVED** [review]

**Pull request merged**